modules/ref-dashboard-configuration-options.adoc

```diff
@@ -68,41 +83,61 @@ On the *Deploy model* dialog when using the single-model serving platform:
-| `spec.dashboardConfig.disableNIMModelServing` | `false` | Enables the ability to select NVIDIA NIM as a model-serving platform. To disable this ability, set the value to `true`.
-| `spec.dashboardConfig.disablePerformanceMetrics` | `false` | Shows the *Endpoint Performance* tab on the *Model deployments* page. To hide this tab, set the value to `true`.
-| `spec.dashboardConfig.disablePipelines` | `false` | Shows the *Data science pipelines* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableProjects` | `false` | Shows the *Data science projects* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
-| `spec.dashboardConfig.disableServingRuntimeParams` | `false` | Shows the *Configuration parameters* section in the *Deploy model* dialog and the *Edit model* dialog when using the single-model serving platform. To hide this section, set the value to `true`.
-| `spec.dashboardConfig.disableStorageClasses` | `false` | Shows the *Settings → Storage classes* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableSupport` | `false` | Shows the *Support* menu item when a user clicks the Help icon in the dashboard toolbar. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableModelServing` | `false` | Shows the *Models → Model deployments* menu item in the dashboard navigation menu, and the *Models* tab in data science projects. To hide these items, set the value to `true`. 
+| `spec.dashboardConfig.` +
+`disableNIMModelServing` | `false` | Enables the ability to select NVIDIA NIM as a model-serving platform. To disable this ability, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disablePerformanceMetrics` | `false` | Shows the *Endpoint Performance* tab on the *Model deployments* page. To hide this tab, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disablePipelines` | `false` | Shows the *Data science pipelines* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableProjects` | `false` | Shows the *Data science projects* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableProjectScoped` | `false` | Distinguishes between global items and project-scoped items (if project-scoped items exist) in the {productname-short} web console. This option applies to workbench images, hardware profiles, accelerator profiles, and model-serving runtimes for KServe. To disable this functionality, set the value to `true`.
```

Comment:
One final doubt. We realised that this feature is fully supported for cluster-admins and supported for workbench images and model-serving runtimes for non-admins. Maybe no - We have already logged a [known issue](https://issues.redhat.com/browse/RHOAIENG-26758) for this. There's a workaround tho so maybe we are good 🤔

Reply:
Yeah this is good to go. Thanks